packages/game-bridge/src/index.ts

```diff
@@ -78,13 +83,36 @@ declare function blu_event(event: string, data: string): void;
 // eslint-disable-next-line @typescript-eslint/naming-convention
 declare function UnityPostMessage(message: string): void;
 
-const callbackToGame = (data: object) => {
+// Type for common callback data
+type CommonCallbackData = {
+  responseFor: string;
+  requestId: string;
+  success: boolean;
+  error: string | null;
+  errorType?: string | null;
+  result?: any;
+  accounts?: string[];
```

Comment:
accounts is not comment too, only used by request accounts. Should it be another type?

Reply:
Added another type for request accounts